.github/workflows/kind-e2e.yml

```diff
@@ -24,7 +21,7 @@ jobs:
         working-directory: ./e2e
         run: ./setup_cluster.sh
 
-      - name: Test: simple
```

Comment:
This colon is intentional to add other tests, such as 'Test: IPv6' or some, so please keep current name.

Reply:
That colon produces a bad syntax error in the yaml file:

https://github.com/zeeke/multi-networkpolicy-iptables/actions/runs/3266782583/workflow

If you really need it, I can change it to 

```
- name: "Test: simple"
```